queue.tf

```diff
@@ -36,48 +36,71 @@ module "processed-envelopes-queue" {
   lock_duration       = "PT5M"
 }
 
+module "payments-queue" {
+  source              = "git@github.com:hmcts/terraform-module-servicebus-queue.git"
+  name                = "payments"
+  namespace_name      = "${module.queue-namespace.name}"
+  resource_group_name = "${azurerm_resource_group.rg.name}"
+  lock_duration       = "PT5M"
+  max_delivery_count  = "${var.payment_queue_max_delivery_count}"
+}
+
 # region connection strings and other shared queue information as Key Vault secrets
 resource "azurerm_key_vault_secret" "envelopes_queue_send_conn_str" {
-  name      = "envelopes-queue-send-connection-string"
-  value     = "${module.envelopes-queue.primary_send_connection_string}"
-  vault_uri = "${data.azurerm_key_vault.key_vault.vault_uri}"
+  name          = "envelopes-queue-send-connection-string"
+  value         = "${module.envelopes-queue.primary_send_connection_string}"
+  key_vault_id  = "${data.azurerm_key_vault.key_vault.vault_uri}"
```

Comment:
all these key vault changes should be ideally a separate PR ?

Reply:
ideally, but it's ok here atm